specs/features/Inconsistencies.md

```diff
@@ -0,0 +1,126 @@
+
+### Rendering
+
+Rendering presents another avenue by which inconsistencies can sneak into Adaptive Cards. Some of these problems can be pretty tricky to solve.
+
+* Multiple implementations means that "gotcha" scenarios might not always be covered or may be covered inconsistently (e.g. what happens when you combine `maxLines`, markdown lists, and `wrap: true`?)
+* Platform-provided controls can vary widely in their behaviors and support. Even if divergent behavior is caught during implementation, it may be difficult or tricky to solve completely
+* Some 
+
+## Proposals
+
+* Attempt to reduce code duplication:
+  * Experiment with the shared model as Javascript (e.g. via [emscripten](https://emscripten.org/))
+  * Move .NET from using Newtonsoft.JSON to using the shared model via P/Invoke (Note that this will have impacts on partners. It's likely that this move will only make sense if Javascript is also using the shared model)
+* Changes processes to prevent new inconsistencies from ever happening:
+  * New bugs automatically filed with `inconsistency` tag, which is only removed once it's determined that the issue is *not* a consistency issue
+  * We can't ship if an `inconsistency`-tagged bug hasn't been resolved for this release
+  * Features that need implementation across all platforms are merged to a single branch and PR'd/tested for inconsistencies before being allowed to merge upstream
+* Provide a new test tool that allows easy testing of card payloads across all renderers (web based? Windows? needs to incorporate all renderers)
+* Parsing/rendering documentation update pass -- leave no ambiguity
+* Blog posts about inconsistencies, how we're addressing them, and how to avoid them when implementing custom parsing/rendering or implementing an entirely new stack
+* Make hard decisions around defaults, required properties, and parser/renderer behaviors
+  * Document them publicly
+  * Blog about them
+  * Blog about philosophy around decisions (enabling third parties to make Adaptive Card-consistent decisions)
+  * Ensure test cards cover scenarios
+  * Make sure we only break our customers once
+* Inconsistency-focused bug bash
+
+## Specific bugs
+
+[List of inconsistency bugs](https://github.com/microsoft/AdaptiveCards/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3AInconsistency+):
+
+* [3605](https://github.com/microsoft/AdaptiveCards/issues/3605) - [Android][Rendering Discrepancy between Android and web] [For Choice sets with compact style, when no option is explicitly pre-selected, first option gets pre-selected]
+* [3573](https://github.com/microsoft/AdaptiveCards/issues/3573) - [.NET C#][Input.ChoiceSet] [adaptive card dropdown title with double quotes rendering issue]
+* [3572](https://github.com/microsoft/AdaptiveCards/issues/3572) - [Web][Avatars] Person avatars are not always circular
```

Comment:
It would be nice to bucketize these issues and include them in the respective inconsistency bucket they belong in above that you are identifying as categories